modules/ROOT/pages/prereq-aws-terraform.adoc

```diff
@@ -38,23 +39,25 @@ This instance runs the actual provisioning of the cluster. You must use an AMI t
 +
 You can also run provisioner remotely from any other machine with Docker and internet access.
 
-. Download the Private Cloud Provisioner (PCP) Docker image from
-+	https://anypoint-anywhere.s3.amazonaws.com/pcp/private-cloud-provisioner-2.1.8.tar.gz?Expires=1753834189&Signature=hjdyrNH84NoiyH1B33elIfCbNbk%3D&AWSAccessKeyId=AKIAI6LBJNQFR2V3CQWQ
+. Download the Private Cloud Provisioner (PCP) Docker image from:
++
+`+https://anypoint-anywhere.s3.amazonaws.com/pcp/private-cloud-provisioner-3.0.16.tar.gz?AWSAccessKeyId=AKIAI6LBJNQFR2V3CQWQ&Signature=gg3SLK48UjBX6qC8iB2Rj3bN1d0%3D&Expires=1765246802+`
```

Comment:
Change to the following link

https://anypoint-anywhere.s3.amazonaws.com/pcp/private-cloud-provisioner-3.0.17.tar.gz?Expires=1766891270&Signature=X05ZzS4jQjGkgFgA6eJS03J5CCY%3D&AWSAccessKeyId=AKIAI6LBJNQFR2V3CQWQ

Reply:
Updated URL.